backends/rapidpro/backend.go

```diff
@@ -174,22 +174,16 @@ func (b *backend) Start() error {
 		s3config := &storage.S3Options{
 			AWSAccessKeyID:     b.config.AWSAccessKeyID,
 			AWSSecretAccessKey: b.config.AWSSecretAccessKey,
+			Region:             b.config.AWSRegion,
 			Endpoint:           b.config.S3Endpoint,
-			Region:             b.config.S3Region,
-			DisableSSL:         b.config.S3DisableSSL,
```

Comment:
Best I can tell this isn't needed if you're providing an endpoint as a full URL with http or https